rpc/rpc-wrappers.go

```diff
@@ -670,6 +671,26 @@ func (pub PublisherClient) SubmitToCT(ctx context.Context, der []byte) (err erro
 	return
 }
 
```

Comment:
@rolandshoemaker I think the better thing to do here is to remove all of the `rpc/rpc-wrappers.go` content I added for `SubmitToSingleCT`. @jsha was on-board with only implementing `SubmitToSingleCT` in gRPC - the OCSP updater is the only consumer of `SubmitToSingleCT` and since https://github.com/letsencrypt/boulder/pull/2341 it won't use AQMP to talk to the Publisher.

Does that seem amenable? I don't think it's worth adding rpc wrapper code that we know won't be used.